src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+const ADD_BOOK = 'bookstore/src/redux/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/src/redux/books/REMOVE_BOOK';
+
+const init = [];
+
+export default function booksReducer(state = init, action) {
+  switch (action.type) {
+    case ADD_BOOK: {
+      const obj = {
+        id: state.length + 1,
+        author: action.newBook.author,
+        book: action.newBook.book,
+
+      };
+      return [...state, obj];
+    }
+    case REMOVE_BOOK: {
+      const newState = state.filter((book) => book.id !== action.id);
+      return newState;
+    }
+
+    default:
+      return state;
+  }
+}
```

Comment:
- [ ] [Optional] It would be better if you used arrow syntax for the reducer function instead of regular syntax.